lib/repository/genre_repo.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+import '../constants/constants.dart';
+import './interface/genre_interface.dart';
+
+class GenreRepository extends GenreInterface {
+  late Map<int, String> genresById = {};
+
+  static Future<Map<int, String>> parseJsonGenres() async {
+    final jsonPath = await rootBundle.loadString(Constants.jsonGenresLocation);
+    Map<String, dynamic> json = jsonDecode(jsonPath);
+    return {
+      for (var genre in json['genres']) genre['id'] as int: genre['name']
+    };
+  }
+
+  @override
+  Future<Map<int, String>> getGenre() async {
```

Comment:
Just curious. Why do you return a `Map<int, String>` instead of a `List<Genre>`?

Reply:
I was using the same for movie_repo, then forgot to replace it in the genres repo too.